packages/vm/src/compatibility.rs

```diff
@@ -86,12 +94,48 @@ fn check_wasm_memories(module: &Module) -> VmResult<()> {
     Ok(())
 }
 
+fn check_interface_version(module: &Module) -> VmResult<()> {
+    let mut interface_version_exports: Vec<String> = module
+        .exported_function_names(Some(INTERFACE_VERSION_PREFIX))
+        .into_iter()
+        .collect();
```

Comment:
Ah, one thing. Do we really need to collect this if we're going to `pop` anyway? May as well keep the iterator and use `next`.

Reply:
Done in b8072de